packages/content-source-maps/src/__tests__/csm.spec.ts

```diff
@@ -272,13 +337,13 @@ describe('Content Source Maps', () => {
           postCollection: {
             items: [
               {
-                title: 'Title of the post',
+                title: 'Title of the first post',
               },
               {
-                title: 'Title of the post 2',
+                title: 'Title of the second post',
               },
               {
-                title: 'Title of the post 3',
```

Comment:
Interestingly the hidden strings don't get added when there's a number at the end 🤔

Reply:
https://contentful.slack.com/archives/C064S7VSW5D/p1708710243893949